client/package.json

```diff
@@ -24,6 +24,7 @@
     "@angular/platform-browser": "~11.2.1",
     "@angular/platform-browser-dynamic": "~11.2.1",
     "@angular/router": "~11.2.1",
+    "component": "^1.1.0",
```

Comment:
Did you mean to add this library? It may have been a typo—did you maybe type `npm install component` instead of `ng generate component`? 🙂

If you don't happen to be using this library for anything, you can run `npm uninstall component` to remove it! :)

Reply:
Oh, never mind, looks like you already took care of this :)